rocketpy/Dispersion.py

```diff
@@ -677,9 +677,6 @@ def run_dispersion(
                     noise=setting["parachutes"][chute]["noise"],
                 )
 
-            # TODO: Fix rail buttons definition here
-            # rocket_dispersion.setRailButtons()
```

Comment:
Ok, @MateusStano , but now when the rail buttons are being added to the rocket within the monte carlo loop?

Reply:
They are being defined just a few lines above